force-app/main/default/layouts/Account-Household Layout.layout-meta.xml

```diff
@@ -29,6 +29,10 @@
                 <behavior>Edit</behavior>
                 <field>npo02__Informal_Greeting__c</field>
             </layoutItems>
+            <layoutItems>
+                <behavior>Edit</behavior>
+                <field>All_Members_Deceased__c</field>
```

Comment:
We aren't including this in our default Contact Layout, so I think it's worth a quick discussion if we want it add here (and exactly where to put it in the Layout!)

Reply:
It might be good to pull it from this PR and add it in either of the next 2 sprints if we decide we want it.